### PR TITLE
Add TestBase and reformatted files

### PR DESCRIPTION
**Pull request by foodie-dev-hongjun2**

### CODE REVIEW

**Comment by foodie-dev-hongjun1** [comment]
Test failed. Can you run locally to check it?😂

**Comment by foodie-dev-hongjun2** [comment]
(Foodie) ➜  FoodiePlatformService git:(dev) python manage.py test
Creating test database for alias 'default'...

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
Destroying test database for alias 'default'...

**Comment by foodie-dev-hongjun1** [comment]
0 tests? Weird... Currently AWS is experience issues and I cannot see testing logs..

**Comment by foodie-dev-hongjun2** [comment]
Yeah.. Looks like some python version mismatch..
http://stackoverflow.com/questions/29358403/no-module-named-urllib-parse-how-should-i-install-it

**Comment by foodie-dev-hongjun1** [comment]
hmm python version issue. Seems like Circle used python 2.X to build, which is not right...

**Comment by foodie-dev-hongjun1** [comment]
It is using python2.7 lib

`Requirement already satisfied: dj-database-url==0.4.2 in /home/ubuntu/virtualenvs/venv-system/lib/python2.7/site-packages (from -r requirements.txt (line 1))
Requirement already satisfied: Django==1.10 in /home/ubuntu/virtualenvs/venv-system/lib/python2.7/site-packages (from -r requirements.txt (line 2))
Requirement already satisfied: django-rest-framework-social-oauth2==1.0.5 in /home/ubuntu/virtualenvs/venv-system/lib/python2.7/site-packages (from -r requirements.txt (line 3))`

**Comment by foodie-dev-hongjun2** [comment]
yeah.. from where did we config CircleCI to build our code?

**Comment by foodie-dev-hongjun1** [comment]
you need to login with foodie admin account and then go to setting tab. But I kind of feel it's not Circle config issue, as it worked fine before.

**Comment by foodie-dev-hongjun1** [comment]
maybe we can try this? 
https://circleci.com/docs/1.0/language-python/

**Comment by foodie-dev-hongjun2** [comment]
I added "python --version" command as a test command and it does print out Python 2.7.11 (which overrided the original django test)
We need to upgrade to use python3 instead. (Ubuntu 14.04 has python3 installed already)

**Comment by foodie-dev-hongjun2** [comment]
Added circle.yml to specify python version. (3.6.1)

**Comment by foodie-dev-hongjun2** [comment]
Fixed :)

**Comment by foodie-dev-hongjun1** [comment]
👍👍👍

File: FoodieApp/tests/test_base.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    SIGNUP_URL = 'cook-signup'
+    COOK_HOME_URL = 'cook-home'
+
+    # registration info
+    DEFAULT_EMIAL = 'test@example.com'
+    DEFAULT_PASSWORD = 'password'
+    WRONG_EMAIL_FORMAT = 'test'
+
+    # default pages
+    DEFAULT_SIGNUP_PAGE = 'base_signup.html'
```
Comment (foodie-dev-hongjun1) [inline FoodieApp/tests/test_base.py]: I would like this to be in views test as no such usage in models and forms. But it can be fixed in next release.

Comment (foodie-dev-hongjun2) [inline FoodieApp/tests/test_base.py]: Yeah, i agree :) updated

**Pull request merged**